parquet/src/main/scala/magnolify/parquet/Schema.scala

```diff
@@ -38,8 +38,8 @@ private object Schema {
       val p = schema.asPrimitiveType()
       Types
         .primitive(p.getPrimitiveTypeName, schema.getRepetition)
-        .length(p.getTypeLength)
         .as(schema.getLogicalTypeAnnotation)
+        .length(p.getTypeLength)
```

Comment:
Does this order matter?

Reply:
No, it a builder, where `named` is the finalizes the result. Just changed order for consistency in all API